monai/handlers/clearml_handlers.py

```diff
@@ -0,0 +1,192 @@
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from .tensorboard_handlers import TensorBoardImageHandler, TensorBoardStatsHandler
+
+
+class ClearMLHandler:
+    """
+
+    Base class for the handlers to log everything to ClearML WebUI.
+
+    Args:
+        project_name:               The name of the project in which the experiment will be created.
+                                    If the project does not exist, it is created. If project_name is
+                                    None, the repository name is used. (Optional)
+        task_name:                  ClearML task name. Default set to 'monai clearml example'.
+        output_uri:                 The default location for output models and other artifacts.
+                                    If True, the default files_server will be used for model storage.
+                                    In the default location, ClearML creates a subfolder for the output.
+                                    The subfolder structure is the following:
+                                    <output destination name> / <project name> / <task name>.<Task ID>.
+                                    Default set to 'True'
+        tags:                       Add a list of tags (str) to the created Task.
+                                    For example: tags=['512x512', 'yolov3'].
+                                    Default set to None.
+        reuse_last_task_id:         Force a new Task (experiment) with a previously used Task ID,
+                                    and the same project and Task name. Default set to 'True'.
+        continue_last_task:         Continue the execution of a previously executed Task (experiment).
+                                    Default set to 'False'.
+        auto_connect_frameworks:    Automatically connect frameworks This includes patching MatplotLib,
+                                    XGBoost, scikit-learn, Keras callbacks, and TensorBoard/X to
+                                    serialize plots, graphs, and the model location
+                                    to the ClearML Server (backend), in addition to original output destination.
+
+    For more details of ClearML usage, please refer to: https://clear.ml/docs/latest/docs/references/sdk/task
+
+    """
+
+    def __init__(
+        self,
+        project_name: str,
+        task_name: str,
+        output_uri: str,
+        tags,
+        reuse_last_task_id: bool,
+        continue_last_task: bool,
+        auto_connect_frameworks,
+    ):
+        try:
+            from clearml import Task
```

Comment:
Please use MONAI `optional_import` for 3rd party packages, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/tensorboard_handlers.py#L35

Thanks.

Reply:
Done !